Imagine/Cache/Resolver/FlysystemResolver.php

```diff
@@ -132,7 +141,7 @@ public function store(BinaryInterface $binary, $path, $filter)
         $this->flysystem->put(
             $this->getFilePath($path, $filter),
             $binary->getContent(),
-            ['visibility' => $this->visibility]
```

Comment:
looks like this caught another use of short array syntax which means this isn't tested